src/pages/Environments.js

```diff
@@ -306,10 +302,9 @@ const Environments = () => {
   }
 
   const renderDeleteDiskModal = disk => {
-    const diskName = disk.name
     return h(DeleteDiskModal, {
       disk,
-      isGalaxyDisk: _.some({ name: diskName }, galaxyDisks),
+      isGalaxyDisk: getDiskAppType(disk) === tools.galaxy.appType,
```

Comment:
The reason we were searching before was that the disk passed in was from the collection of all disks, and the only way we had to know if it was a Galaxy disk was to see if the disk also existed in the smaller galaxyDisks collection.